.github/workflows/sdk-regression.yml

```diff
@@ -21,19 +21,30 @@ jobs:
           - percy-nightwatch
           - percy-nightmare
           - percy-webdriverio
+          - percy-webdriverio@v2
           - percy-protractor
           - percy-selenium-js
           - percy-selenium-dotnet
           - percy-selenium-java
+          - percy-selenium-python
+          - percy-capybara
+          - percy-appium-js
           - gatsby-plugin-percy
     steps:
+      - uses: jungwinter/split@v2
+        id: split
+        with:
+          msg: ${{ matrix.repo }}
+          separator: '@'
+      - uses: xt0rted/pull-request-comment-branch@v2
+        id: comment-branch
       - name: Trigger Workflow & Wait
         uses: convictional/trigger-workflow-and-wait@v1.6.5
         with:
           owner: percy
           repo: ${{ matrix.repo }}
           github_token: ${{ secrets.WORKFLOW_DISPATCH_ACTIONS_TOKEN }}
           workflow_file_name: test.yml
-          ref: master
-          client_payload: '{ "branch": "${{ github.event.pull_request.head.ref || github.ref_name }}"}'
+          ref: ${{ steps.split.outputs._1 || master }}
+          client_payload: '{ "branch": "${{ steps.comment-branch.outputs.head_ref || github.ref_name }}"}'
```

Comment:
I missed this before, why do we need `github.ref_name` this? we should fail the build if `comment-branch` fails, we don't want incorrect refs to trigger the build and have the checks green.